### ISSUE TITLE
The dataset 

### ISSUE THREAD

**Issue by pengchengwanlixulipeng** [oshadajay/CeyMo#7]
The json file in the dataset has "imageData": null, which prevents it from being converted to COCO format.

**Comment by oshadajay** [comment]
Hi, could you please attach the script you are using for the conversion?

**Comment by pengchengwanlixulipeng** [comment]
Thanks, I got the Imagedata by "saving as" with labelme, but I have another problem, how many epochs to train this dataset on Mask-R-CNN to reach the best state, as I finished training before 20 epochs, I am confused about it. @oshadajay

**Comment by oshadajay** [comment]
Cool. We used the Tensorflow Object Detection API and we did not trained the model for a fixed number of epochs. We observed the evaluation metrics which are published from time to time and stopped training when the metrics sort of saturated. Hope this helps.

**Comment by pengchengwanlixulipeng** [comment]
Thanks, my loss function was fitted at 20 epoch, I think the reason for this is related to the low diversity of the dataset, after all, many images taken at adjacent times in the dataset have similarities, so the model learns the features of the dataset very quickly.

**Comment by oshadajay** [comment]
Yeah that could be the reason.
